### PR TITLE
Iframe transport documentation

### PR DESCRIPTION
**Pull request by jonkeller**
Document iframe transport

### CODE REVIEW

**Comment by jonkeller** [comment]
Do we know yet where https://docs.google.com/document/d/1Rg2yDDG4tHBe-_I4iJZKCJ_YkotjvbSBaf1CfZjpI2s/edit?usp=sharing is going to live? It would be nice to put a link to that.

File: extensions/amp-analytics/amp-analytics.md [hunk 1]
```diff
@@ -521,6 +523,8 @@ In the example below, `beacon` and `xhrpost` are set to `false`, so they will no
 }
 ```
 
+If the `iframe` transport option is used, the specified URL must point to a page which implements the iframe transport client API. See the [example](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport-remote-frame.html) of how to use this API, and also an [example](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport.amp.html) of a page that incorporates that iframe. The example loads a [fake ad](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html), which contains the `amp-analytics` tag. Note that the fake ad content includes some extra configuration instructions that must be followed.
```
Comment (bpaduch) [inline extensions/amp-analytics/amp-analytics.md]: Consider rewriting "See ..." so the links are a wee bit more specific, like so:

`To learn more, see [this example that implements iframe transport client API](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport-remote-frame.html) and [this example page that incorporates that iframe](https://github.com/ampproject/amphtml/blob/master/examples/analytics-iframe-transport.amp.html). The example page ...`

File: extensions/amp-analytics/why-not-iframe.md [hunk 2]
```diff
@@ -14,3 +14,5 @@ With AMP, The AMP Project decided to encourage the use of the AMP Analytics syst
 The disadvantages of this approach are:
 * **Less data points (initially):** Initially, the `amp-analytics` extension's data collection will not be as complete as existing systems. This is being addressed by multiple partners in the Github repo. The AMP Project is committed to enhancing the capabilities of amp-analytics based on the foundation that’s been built thus far.
 * **Adaption to a new technical approach:** For example, you may use a client-side collection technique that needs to be substituted for a server-side solution.
+
+If this is insufficient, MRC-accredited vendors may still use an embedded iframe. This is discussed in the [amp-analytics documentation](amp-analytics.md)
```
Comment (bpaduch) [inline extensions/amp-analytics/why-not-iframe.md]: Add period.

**Comment by jonkeller** [comment]
Thanks!

**Pull request merged**